src/posts/features-and-projects.md

```diff
@@ -0,0 +1,146 @@
+
+---
+
+## Feature Updates
+
+### Display Typeface Development
+
+We believe in a holistic approach to aircraft development - we do not sacrifice one aspect of the aircraft for another, and this means we put as much attention into visuals as we do into systems. As part of this quest for visual accuracy, we have spent countless hours creating custom typefaces that replicate the look of Airbus’ many display fonts. At last count, there are five cockpit display typefaces on both the A320 and A380 - the EIS font, the ISIS font, the FMS font (A320 only), the DCDU font (A320 only) and the RMP font (A380 only). This does not include the seven-segment displays dotting the cockpit panels as well as the A380 EFIS control panel LCDs.
+
+> How do we do them, and where do we start?
+
+First, we determine whether the font is anti-aliased. If it is, we look for head-on closeups of the glyphs (letters, numbers and symbols) and carefully trace each glyph. If not, we look for underexposed display shots which allow us to discern every pixel on the glyphs, then trace the outlines of each glyph (the ‘path’) to create a pixel-perfect font. Once we are done with this, we determine the amount of spacing between letters, which in this case is consistent for all characters within each of the fonts since all of aforementioned fonts are monospaced.
+
+Next, we export them as SVGs into a font editing software. Here, we adjust the size and spacing of the glyphs, as well as fix font formatting errors (e.g. path self-intersections, missing extremas, path direction issues and non-integral point values). Finally, we export them as .ttf files for use on the sim aircraft’s displays.
+
+The following is a summary of the progress we have made:
+
+- The EIS font has been merged into the aircraft
+- The ISIS font is in the works
+- The DCDU fonts are being reworked
+- The A380 RMP fonts are in advanced progress 
+ 
+Here are some previews of the ISIS and RMP fonts:
+
+![typeface 1](/img/notam-images/march2022/typeface-1.png)
+![typeface 2](/img/notam-images/march2022/typeface-2.png)
+
+One last thing: we rework our fonts as new information and references come, some of which have fundamentally changed our understanding of the displays’ workings (e.g. subpixel array design). As an example, here are photos of our DCDU font before and after reworking:
+
+![typeface 3](/img/notam-images/march2022/typeface-3.png)
+![typeface 4](/img/notam-images/march2022/typeface-4.png)
+
+In this case, it has allowed us to go one step further and reach the final frontier in display typeface replication - previously uncharted territory in the flight simulation industry - sub-pixel accuracy.
```

Comment:
```suggestion
In this case, it has allowed us to go one step further and reach the final frontier in-display typeface replication - previously uncharted territory in the flight simulation industry - sub-pixel accuracy.
```

Reply:
no dash, it's not supposed to say 'in-display'

Reply:
Also leaving this untouched.